test/test_utilities.cpp

```diff
@@ -70,6 +70,11 @@ void PrintTensor(const Tensor& T1) {
   LOG(INFO) << "print tensor values" << T1.DebugString();
 }
 
+// Only displays values in tensor without shape information etc.
+void PrintTensorAllValues(const Tensor& T1, int64 max_entries) {
+  LOG(INFO) << "all tensor values" << T1.SummarizeValue(max_entries) << endl;
+}
+
```

Comment:
This is an useful function.
ideally this function would accept a default val of `max_entries`, set to -1
when it sees `-1`, it would compute the total num of elements in the tensor and display it.

But we dont have to do this in this PR